include/silo/storage/sequence_store.h

```diff
@@ -77,4 +84,10 @@ class SequenceStore {
 
 }  // namespace silo
 
+template <>
+struct fmt::formatter<silo::SequenceStoreInfo> : fmt::formatter<std::string> {
```

Comment:
My Clion says, that we can mark this as `maybe_unused`. I like that because it tells us later on, that there is a purpose behind this entry. :)